### ISSUE TITLE
Not getting any output from example app even though run shows servers

### ISSUE THREAD

**Issue by tonym128** [Nimrodda/flutter_nsd#32]
I'm running the example app on windows and android
https://github.com/Nimrodda/flutter_nsd/tree/master/example

And I can see debug ouput, but nothing is ever posted to the listener

I can see this in my run output
192.168.101.100:5353 : answer osmc.local. A 192.168.101.100
[fe80::a72e:a506:d95d:fae7%65]:5353 : answer _device-info._tcp.local. PTR README._device-info._tcp.local. rclass 0x1 ttl 10 length 9

But flutterNsd.stream.listen never seems to be called

I've tried "_http._tcp." and "_device-info._tcp." which both return this server, but the services array and the UI are never updated to reflect that.

Any help would be appreciated

**Comment by Nimrodda** [comment]
This seems like a local issue on your side. I just tried both Windows and Android example clients while running a fake server on MacOS with `dns-sd -R TestService _http._tcp . 3000` and could successfully see the service as can be seen from the screenshot below:

![image](https://user-images.githubusercontent.com/149684/154971819-fe321c2f-826d-43c1-a658-772e253f2190.png)

Which means `flutterNsd.stream` emits services as they're discovered.

What version of Flutter are you on?
Are you using the latest version of flutter_nsd?

A few tips on how to troubleshoot the issue: 

1. Try the pre-built example clients from Github:
Windows: https://github.com/Nimrodda/flutter_nsd/suites/5226672395/artifacts/160346656
Android: https://github.com/Nimrodda/flutter_nsd/suites/5226672392/artifacts/160345440
The pre-built clients are configured to discover services of type: `_http._tcp.` for example a HP Wifi Printer should appear be discovered.

2. Microsoft has a command line tool to discover services: https://github.com/microsoft/FindDevice/releases/tag/v1.0.0
Example command: `FindDevice.exe --service _http._tcp.local`

3. If you can't get any service discovered, you can try a fake service like the one I mentioned above from MacOS (`dns-sd`). I bet there's a similar tool for Windows as well.

**Comment by tonym128** [comment]
Thank you for help so far.

I tried running the pre-compiled app and still did not find any services.

I then downloaded FindDevice and ran as suggested, it found 
Discovered: osmc.local 192.168.101.100

Once I started up the dns-sd with the command you provided FindDevices showed
Discovered: {fakeDevice}.local 192.168.101.169
Discovered: osmc.local 192.168.101.100

And the flutter example app was able to find the local fake device from dns sd
![image](https://user-images.githubusercontent.com/1162997/154983010-83a55122-e358-4df3-9907-cb9532ee3cd0.png)

Flutter 2.10.1
flutter_nsd - I ran with a package install first, then currently I'm using the latest version of git, but since there's issues finding the osmc service with the pre-built package, I don't think it's something on my side, maybe just the way the service is presenting itself to NSD.
